deps.edn

```diff
@@ -15,6 +15,10 @@
    :extra-deps {metosin/reitit-frontend {:mvn/version "0.5.15"}
                 org.babashka/sci {:mvn/version "0.2.9"}}}
 
+  :nextjournal/clerk
+  {:exec-fn nextjournal.devdocs/build!
+   :exec-args {:paths [{:path "docs" :filter-pattern "clerk|devcards"}]}}
```

Comment:
Since this isn't calling Clerk code I don't think the `:nextjournal/clerk` alias is right for this. Maybe `:nextjournal/devdocs`?

Reply:
> get out of the business of inventing a small language

true, I looked for the cheapest in flexibility. At present we're mixing inclusion/exclusions like in ductile: 

```
  :nextjournal/clerk
  {:exec-fn ductile.clerk/build!
   :override-deps {io.github.nextjournal/clerk {:git/sha "8b5ff22d8f84cfdb16fb0bfe8a1187d64293f3be"}} ;; Clerk with improved exception info
   :extra-paths ["notebooks"]
   :exec-args {:paths [{:path "notebooks"
                        :exclude-pattern ["vin_scanner"
                                          "reify_simulation_data"]}
                       {:path "roadmap"}
                       {:path "dev/ductile/insights"
                        :exclude-pattern "edi"}
                       {:path "src/re_db"
                        :filter-pattern "notebook"}]}}

```
negations seems more difficult in term of globs?

> letting folks specify a symbol

would this target a single notebook? Arguing by a (collection of) top folder paths makes it easier I think...

Reply:
In the above "patterns"  are converted in `some-fn` (for exclude) and `every-pred` (for filter) of `re-find` functions where strings are converted via `re-pattern` but regex literals are also accepted.